src/pytest_cov/engine.py

```diff
@@ -221,6 +227,9 @@ def finish(self):
 
         # Combine all the suffix files into the data file.
         self.cov.stop()
+        self.cov.save()
+        self.cov = self.combining_cov
+        self.cov.load()
```

Comment:
I'm not sure this is needed in a distributed mode, unless you're having each worker report itself (which I think is a feature you have). It probably would cause problems in the normal case, though, since I think the parallel version of the coverage file gets deleted normally when it's being combined.